bindings/python/iota_sdk/utils.py

```diff
@@ -58,6 +58,15 @@ def account_id_to_bech32(account_id: HexStr, bech32_hrp: str) -> str:
             'bech32Hrp': bech32_hrp
         })
 
+    @staticmethod
+    def anchor_id_to_bech32(anchor_id: HexStr, bech32_hrp: str) -> str:
```

Comment:
not optional here?

Reply:
No since there is no client to get it if not provided